mesh2hrtf/NumCalc/manage_numcalc.py

```diff
@@ -418,16 +415,22 @@ def _raise_error(message, text_color, log_file, confirm_errors):
 
     # error to console
     if confirm_errors:
-        print(text_color + message)
-        input(text_color + "Press Enter to exit num_calc_manager\033[0m")
-        raise Exception("num_calc_manager was stopped due to an error")
+        if os.name == 'nt':  # Windows detected
+            print(message)
+            input("Press Enter to exit manage_numcalc")
+        else:  # elif os.name == 'posix': Linux or Mac detected
+            print(text_color + message)
+            input(text_color + "Press Enter to exit manage_numcalc\033[0m")
+        raise Exception("manage_numcalc was stopped due to an error")
     else:
         raise ValueError(message)
 
 
 def _print_message(message, text_color, log_file):
     """Print message to console and log file"""
 
+    if os.name == 'nt':  # Windows detected
+        text_color = ''  # color codes do not work as intended on Win10
```

Comment:
I would be fine with completely removing color output in this case. What do you think?

Reply:
I don't mind to dropping the color support. Only it may be possible to fix this, because the NumCalcManager.py and other scripts that I used previously worked nicely with colors in Windows command prompt, so perhaps there is some trick to fix it?
Up to you, but as it is it does not hurt on Windows and probably does work on Linux?

Reply:
Yes it works on Linux - I'm also fine with leaving is as it is.